Assets/_BForBoss/Tests/CharacterMovementAudioTests.cs

```diff
@@ -0,0 +1,295 @@
+using NUnit.Framework;
+using Perigon.Character;
+using UnityEngine;
+
+namespace Tests.Character
+{
+    public class CharacterMovementAudioTests
+    {
+        private const float MIN_SPEED  = 5f;
+        
+        [Test]
+        public void GetNextStateFromNotPlaying_whenSpeedMagnitudeHigherThanMinSpeed_whenIsWallRunningTrue_resultingStateWallRunning()
+        {
+            // Given
+            var dummyCharacterMovement = new DummyCharacterMovement();
+            var minSpeed = 1;
+            var currentSpeed = 10;
+            dummyCharacterMovement.IsWallRunning = true;
+            dummyCharacterMovement.SpeedMagnitude = currentSpeed;
+            var movementAudio = new CharacterMovementAudio(dummyCharacterMovement, minSpeed);
```

Comment:
Basically, these are the state of the character. And the "next state" is the expected result
```
            var minSpeed = 1;
            var currentSpeed = 10;
            dummyCharacterMovement.IsWallRunning = true;
            dummyCharacterMovement.SpeedMagnitude = currentSpeed;
```